test/SourceKit/Macros/macro_semantic_token.swift

```diff
@@ -0,0 +1,37 @@
+
+@freestanding(declaration)
+macro anonymousTypes(_: () -> String) = #externalMacro(module: "MacroDefinition", type: "DefineAnonymousTypesMacro")
+
+#anonymousTypes { "hello" }
+
+// REQUIRES: swift_swift_parser, executable_test, shell
```

Comment:
Adding `shell` just for the echo seems... not great. What do you need it for?

Reply:
Oh, echo was just a leftover from debugging. I copied `REQUIRES: shell` from 

https://github.com/apple/swift/blob/4ac34a40ea9e3193be342b58ad2e1abb2399a155/test/SourceKit/Macros/macro_basic.swift#L63

I think it’s needed for the `COMPILER_ARGS` shell variable.

Reply:
You can use `-vv` assuming you just wanted the failing line.

And yeah, you're right - it's needed for the `COMPILER_ARGS`. Given this is really only 4 args, I'd prefer just copying them (and it's only twice without the echo).

Reply:
`-vv` didn’t expand the `COMPILER_ARGS` 😢 That’s why I used `echo`… Yet another reason to inline them.

Reply:
Oh I see :(